front-end/js/jump.js

```diff
@@ -330,6 +330,299 @@ function startJump(viewId, d, jump, optionalArgs) {
     else if (jump.type == param.highlight) highlight(predArray, jump);
 }
 
+// send updates to DB
+function doDBUpdate(viewId, canvasId, layerId, tableName, newObjAttrs) {
+    // find field in newObjAttrs that is the primary key identifier (like "id")
+    let idColumns = [];
+    const attributes = Object.keys(newObjAttrs);
+    for (let idx = 0; idx < attributes.length; idx++) {
+        const val = attributes[idx];
+        if (val.includes("id")) {
+            idColumns.push(val);
+        }
+    }
```

Comment:
Need to document what assumptions about SQL query there are. From what I see, 

* It needs to start with "SELECT [fields] FROM tbl".
* [fields] need to include some columns that start with `id`. 
* Changing raw data attributes that affect position will not update the position. The only way to change position right now is to drag and drop. 

What else?

Reply:
That makes sense, those assumptions are a good place to start. Another point to clarify is that updatable attributes that are in the base table are automatically updated. If an attribute is only in the index table, then there's only one update. But if an attribute is in the index table and the base table (like "home_score" and "away_score" in the nba example) then there are technically 2 updates.